### PR TITLE
Made changes to fix errors from issue #406

### PR DESCRIPTION
**Pull request by rahulpatel596**
1. Added async await to fix promise error.
2. Added generic/backup font `font-family:Pacifico, Arial, Helvetica, Sans-serif;`

### CODE REVIEW

**Comment by rahulpatel596** [comment]
@daltonfury42 Hey, would you be able to review this PR. Its failing on  **Lint / Run linters (pull_request)** 
Let me know if I need to make any changes

File: simplq/src/components/common/NotificationButton.jsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ const NotificationButton = () => {
   const dispatch = useDispatch();
   const notificationPermission = useSelector((state) => state.appReducer.notificationPermission);
 
-  const hasPromiseBasedNotificationSupport = () => {
+  const hasPromiseBasedNotificationSupport = async () => {
     try {
-      Notification.requestPermission().then();
+      await Notification.requestPermission().then();
```
Comment (daltonfury42) [inline simplq/src/components/common/NotificationButton.jsx:12]: Do we need the `.then()` here? Else let's remove it.

cc @thehamzarocks

**Review by daltonfury42 — COMMENTED** [review]
Thanks to you for the PR!

File: simplq/src/components/pages/Status/NotificationContainer.jsx [hunk 2]
```diff
@@ -9,13 +9,13 @@ export default () => {
   const dispatch = useDispatch();
   const notificationPermission = useSelector((state) => state.appReducer.notificationPermission);
 
-  const hasPromiseBasedNotificationSupport = () => {
+  const hasPromiseBasedNotificationSupport = async () => {
     try {
-      Notification.requestPermission().then();
+      await Notification.requestPermission();
     } catch (e) {
+      console.error(e);
```
Comment (daltonfury42) [inline simplq/src/components/pages/Status/NotificationContainer.jsx:16]: Remove

**Review by daltonfury42 — APPROVED** [review]
Thanks for the PR.

**Comment by daltonfury42** [comment]
@rahulpatel596 https://github.com/SimplQ/simplQ-frontend/issues/413 is a interesting issue you can work on if you would like to.

**Pull request merged**